### ISSUE TITLE
mubsub causes `retryInterval` error 

### ISSUE THREAD

**Issue by alphara** [scttnlsn/mubsub#56]
I tried to integrate several socket.io to MongoDB adapters to my Node.js application, like:
https://www.npmjs.com/package/socket.io-adapter-mongo
https://www.npmjs.com/package/socket.io-mongodb
https://www.npmjs.com/package/socket.io-adapter-mongo-replica

All of them uses 'mubsub' module as dependency. At the connection, **mubsub** gives error
```
Caught exception: { MongoError: The field 'retryInterval' is not a valid collection option. Options: { capped: true, size: 5242880, retryInterval: 200, recreate: true }
```
Can you fix it, please?

**Comment by scttnlsn** [comment]
This looks like a duplicate of https://github.com/scttnlsn/mubsub/pull/55

**Comment by scttnlsn** [comment]
Merged #55
